planemo/galaxy/config.py

```diff
@@ -659,13 +660,15 @@ def __init__(
         galaxy_url,
         master_api_key,
         user_api_key,
+        noverify,
         runnables,
         kwds,
     ):
         self._ctx = ctx
         self.galaxy_url = galaxy_url
         self.master_api_key = master_api_key
         self._user_api_key = user_api_key
+        self.noverify = noverify,
```

Comment:
```suggestion
        self.verify = not noverify,
```

and you need to add `, verify=self.verify` to the calls to `gi() on lines 679 and 698.

Reply:
ok, thank you. I've done the modifications.